src/lib/ui/AccessCodeDetails/AccessCodeDetails.tsx

```diff
@@ -1,26 +1,16 @@
 import { DateTime } from 'luxon'
 import type { AccessCode } from 'seamapi'
 
-import { useFakeAccessCode } from 'lib/index.js'
-
 import AccessCodeDevice from 'lib/ui/AccessCodeDetails/AccessCodeDevice.js'
 import { ContentHeader } from 'lib/ui/layout/ContentHeader.js'
 
 export interface AccessCodeDetailsProps {
-  accessCodeId: string
+  accessCode: AccessCode
 }
 
 export default function AccessCodeDetails({
-  accessCodeId,
+  accessCode,
```

Comment:
Hmm doesnt seem right

Reply:
It's possible there were some changes with faking the access code data in this revert that will be added back in the updated PR. I think we will resolve it quickly after this is merged.

Reply:
Yep i figured